### ISSUE TITLE
The BarBackgroundColor does not work.

### ISSUE THREAD

**Issue by gentleman12** [MvvmCross/MvvmCross-Samples#132]
The BarBackgroundColor does not work.
BarTextColor works, but I do not know why the background color property of the bar does not work on my navigation page.

I am using the last version of mvvmcross.forms and verified the starwars sample, but for me the background color of the bar does not work.

Does anyone have the same problem?

**Comment by nmilcoff** [comment]
Sorry, I'm not sure I understood. Is this working on the StarWarsSample?

**Comment by gentleman12** [comment]
Yes, in starwarsample the barbackgroundcolor property but when I use the same style for my project, barbackgroundcolor no work

<Application.Resources>
        <ResourceDictionary>
            <Style TargetType="NavigationPage">
                <Setter Property="BarBackgroundColor" Value="Green"/>
                <Setter Property="BarTextColor" Value="Green"/>
            </Style>
        </ResourceDictionary>
    </Application.Resources>
on this style only work BarTextColor in my navigation page

**Comment by gentleman12** [comment]
Sorry, I found the solution, the problem was that I deleted ToolbarResource = Resource.Layout.Toolbar; from my rootactivity.

again apologize
